android-database-sqlcipher/src/main/java/net/sqlcipher/database/SQLiteDatabase.java

```diff
@@ -54,27 +57,28 @@
 import android.util.Log;
 import android.util.Pair;
 
+import java.io.UnsupportedEncodingException;
+
 /**
- * Exposes methods to manage a SQLite database.
+ * Exposes methods to manage a SQLCipher database.
  * <p>SQLiteDatabase has methods to create, delete, execute SQL commands, and
  * perform other common database management tasks.
- * <p>See the Notepad sample application in the SDK for an example of creating
- * and managing a database.
+ * <p>A call to <code>loadLibs(…)</code> should occur before attempting to
+ * create or open a database connection.
  * <p> Database names must be unique within an application, not across all
  * applications.
  *
- * <h3>Localized Collation - ORDER BY</h3>
- * <p>In addition to SQLite's default <code>BINARY</code> collator, Android supplies
- * two more, <code>LOCALIZED</code>, which changes with the system's current locale
- * if you wire it up correctly (XXX a link needed!), and <code>UNICODE</code>, which
- * is the Unicode Collation Algorithm and not tailored to the current locale.
  */
 public class SQLiteDatabase extends SQLiteClosable {
     private static final String TAG = "Database";
     private static final int EVENT_DB_OPERATION = 52000;
     private static final int EVENT_DB_CORRUPT = 75004;
+    private static final String KEY_ENCODING = "UTF-8";
 
-    public static final String SQLCIPHER_ANDROID_VERSION = "3.4.0-os.2";
+  /**
+   * The version number of the SQLCipher for Android Java client library.
+   */
+    public static final String SQLCIPHER_ANDROID_VERSION = "3.5.9";
```

Comment:
Is this PR updating the SQLite version?

Reply:
No, first line: This PR includes ONLY the support of 64-bits libraries (provided by upstream tag 3.5.9). As I said in the old PR and in the Slack channel, during the plugin certification, while upgrading an app using Cipher Local Storage 3.8.10.2 or 3.5.9 to version 4.X, resulted in an inaccessible database after the upgrade. Due to the due date imposed by Google and the pressure from the support team, we will only release the support for 64-bits first and then we'll tackle the issue later in *RNMT-2950* (https://outsystemsrd.atlassian.net/browse/RNMT-2950).